plugin/src/main/java/io/jenkins/plugins/casc/core/UpdateSiteConfigurator.java

```diff
@@ -1,17 +1,23 @@
-package io.jenkins.plugins.casc.plugins;
+package io.jenkins.plugins.casc.core;
```

Comment:
shouldn't this belong in the support plugin?
its only here because core is missing the right attributes

Reply:
Update center is apart of Jenkins Core model so my bet would be no :blush:

Would be much better if they were made Databound. 

But we can definitely move it to support but I would need to know how I can add it to the JCasC model, cause it does. NOT appear on it's own 😭

Reply:
hmm looks a lot easier to leave it where it is for now